heat/core/manipulations.py

```diff
@@ -3537,34 +3537,15 @@ def resplit(arr: DNDarray, axis: int = None) -> DNDarray:
             gathered, is_split=axis, device=arr.device, comm=arr.comm, dtype=arr.dtype
         )
         return new_arr
-    arr_tiles = tiling.SplitTiles(arr)
+
```

Comment:
Should we update the docstring here, and mention the Dalcin paper?

Reply:
Do you mean the FFT paper? I would reference it inside ```Alltoallw```, as much of the relevant code landed there, or in ```tiling.py```

Reply:
OK for referencing inside Alltoallw